automated-demo/auto-demo.sh

```diff
@@ -1075,5 +1075,7 @@ STEPS=${STEPS:="all"}
 MODE=${MODE:="demo"}
 CI_MODE=${CI_MODE:="0"}
 DEMO_NAMESPACE=${DEMO_NAMESPACE:="federation-demo"}
+OPENSHIFT_CONSOLE=${OPENSHIFT_CONSOLE:="openshift-master"}
```

Comment:
The default should be console, isn't it?

Reply:
I think the name can vary. From looking at the docs. You may be correct with the default value being console but looks like sometimes they use openshift 

https://docs.openshift.com/container-platform/3.11/install/example_inventories.html